### ISSUE TITLE
[Shadergraph] Make shadergraph complain about a missing node

### ISSUE THREAD

**Issue by QuackCola** [Facepunch/sbox-issues#5805]
### For?

S&Box

### What can't you do?

At the moment shadergraph doesn't complain about missing nodes when you open up a shadergraph asset. it just doesn't load the graph with the missing node when you try to open up a graph.

### How would you like it to work?

Have a warning dialog popup with the _class of the node that cannot be found. Much like a compiler error when you open up a .sbproj.

### What have you tried?

n/a.

**Comment by MD485** [comment]
Yeah I'd much prefer it load as much of the shader as it can and have some error visualisation rather than it silently failing.
Something like this would be insanely helpful.
![image](https://github.com/Facepunch/sbox-issues/assets/36082767/91701ffa-f975-4b31-ba2d-c1f817cb4140)
But even just a text node without any connections in the ui would be great, but you might have to add a debug view to properties to show stuff like ids for people to make sense of it.
![image](https://github.com/user-attachments/assets/6f1a8e05-dfd3-4576-91a0-8f927fb04864)
